### DISCUSSION TITLE
Row selection in datagrid

### DISCUSSION THREAD

**Discussion by drdotnet22** [microsoft/fluentui-blazor#1263] [Q&A] [answered]
Is there any option for row selection in the datagrid and performing operations on selected rows? I can't find anything in the docs.
I'm looking for something similar to this. https://blazor.syncfusion.com/demos/datagrid/selection?theme=fluent

**Comment by vnbaaij** [comment]
There is the OnRowFocus event Händler that you can hook up to. See https://fluentui-blazor.net/DataGrid#manual grid for an example

**Reply by drdotnet22** [reply]
I should have been more clear. I need multi row selection with the ability to perform an operation on all selected rows. The other datagrid I linked to has checkbox, drag and row clicked options for selecting multiple rows at a time.

**Reply by vnbaaij** [reply, marked as answer]
No worries. No, those are not actions this grid supports. And, considering the amount of work and complexity involved, are not going to be added in the short term (if at all).

**Reply by drdotnet22** [reply]
Thanks for confirming that.
